extensions/interactions/TextInput/directives/text-input-validation.service.spec.ts

```diff
@@ -92,6 +105,9 @@ describe('TextInputValidationService', () => {
   });
 
   it('should catch non-string value for placeholder', () => {
+    // This throws "Type 'number' is not assignable to type 'SubtitledUnicode'".
+    // We need to suppress this error because we need to test validations.
+    // @ts-ignore
```

Comment:
Should we use `@ts-except-error` here? Ditto below

Reply:
Yeah updated here, not required below